src/vcRender.cpp

```diff
@@ -1130,7 +1149,42 @@ bool vcRender_CreateSelectionBuffer(vcState *pProgramState, vcRenderContext *pRe
   }
   return true;
 }
+void vcRenderWatermark(vcState *pProgramState, vcRenderContext *pRenderContext)
```

Comment:
There should be an empty line before this line - functions are separated by empty lines.

Reply:
Fixed.